packages/components/src/internal/components/domainproperties/dataset/DatasetDesignerPanels.spec.tsx

```diff
@@ -38,8 +38,8 @@ describe('Dataset Designer', () => {
     const newDatasetModel = DatasetModel.create(NEW_DATASET_MODEL_WITHOUT_DATASPACE, undefined);
     const populatedDatasetModel = DatasetModel.create(null, getDatasetDesign);
 
-    test('New dataset', () => {
-        const designerPanels = renderer.create(
+    test('New dataset', async () => {
+        const designerPanels = mount(
```

Comment:
same comment here. The snapshot change is too big for me to review here in the github UI. Do you think the test is still checking for the desired outcome?

Reply:
See above